app/src/main/java/com/myperishableplanner/v21001/MainActivity.kt

```diff
@@ -26,20 +21,23 @@ class MainActivity : ComponentActivity() {
         setContent {
             MyPerishablePlannerTheme {
                 // A surface container using the 'background' color from the theme
-                Surface(color = MaterialTheme.colors.background , modifier = Modifier.fillMaxWidth()) {
-                    ExpirationFacts("Android")
+                Surface(
+                    color = MaterialTheme.colors.background,
+                    modifier = Modifier.fillMaxWidth()
+                ) {
+                    expirationFacts("Android")
```

Comment:
Class headers that are longer should be formatted so that each primary constructor parameter is in a separate line and indented per: https://kotlinlang.org/docs/coding-conventions.html#configure-style-in-ide